antenna-documentation/scripts/execute_github_site_maven_plugin.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash -e
```

Comment:
AFAIK this does not work, you can not use `/usr/bin/env` and pass arguments to `bash`. Better add the line `set -e` instead

Reply:
That's how @neubs-bsi actually had it before and I suggested her to change it. `env` actually *does* pass following args to the command, so it should work, see e.g. `env ls -l`. And if it doesn't, I'm to blame, not @neubs-bsi 😉

Reply:
but:
```
$ echo "#!/usr/bin/env ls -l" > test
$ chmod +x test
$ ./test
/usr/bin/env: ‘ls -l’: No such file or directory
/usr/bin/env: use -[v]S to pass options in shebang lines
```

Reply:
Odd, it worked in my quick test. [This answer](https://unix.stackexchange.com/questions/63979/shebang-line-with-usr-bin-env-command-argument-fails-on-linux) suggests that different Unixes behave differently. Bottom line, *if* we use `env`, then indeed `set -e` needs to be separated out, sorry @neubs-bsi 😢 And I remember having had this discussion already the last time when working on shell scripts that use `env`, sorry again for forgetting about it.